csrankings-s.csv

```diff
@@ -1676,7 +1676,6 @@ Steven M. Nowick,Columbia University,http://www.cs.columbia.edu/~nowick,gaXiAekA
 Steven M. Seitz,University of Washington,http://cs.washington.edu/homes/seitz,-6cOYcYAAAAJ
 Steven Mills,University of Otago,http://www.cs.otago.ac.nz/staff/Steven_Mills,lfySPjwAAAAJ
 Steven Minsker,University of Arkansas - Little Rock,https://ualr.edu/computerscience/about/faculty-and-staff/steven-minsker,NOSCHOLARPAGE
-Steven Myers,Indiana University,http://wphomes.soic.indiana.edu/samyers,SQnCsswAAAAJ

```

Comment:
Please add to `old/industry.csv` with affiliation at the end.

Reply:
done